frontend/src/components/script-editor.tsx

```diff
@@ -67,7 +67,7 @@ export default function ScriptEditor() {
       </div>
       <div
         className={clsx(
-          split ? "border-b-4 h-2/5" : "rounded-b-xl h-full",
+          split ? "border-b-4" : "rounded-b-xl h-44",
```

Comment:
I think it would be preffered for the script editor to stay the same size after splitting ( which is why I did the 2/5 & 3/5 thing when split ). And the script editor size should probably be a bit larger than this. What do you think?

Reply:
We'll look into this right now

Reply:
So for some reason even after applying a specific height to the parent the 2/5 and 3/5, so I am currently thinking a solution where we use 60% of 376px and 40% of 376px hardcoded in, that is 150px and 225px respectively, gives the same effect

Reply:
Sounds good, that works for me. 👍

Reply:
Okay boss should have it pushed in bout 30mins, just verifying a few other things

Reply:
@b-j-roberts we have implemented the 150/225px sizes